README.md

```diff
@@ -112,20 +112,6 @@ logger.on('error', (error) => {
 
 See documentation from [docs/configuration](docs/configuration.md)
 
-### How to keep the connection open while Logstash is restarting?
```

Comment:
Since this behavior is now the default, I don't think we need this part of the documentation anymore